test/test_shape_ops.py

```diff
@@ -605,7 +606,19 @@ def test_nonzero_non_diff(self, device):
         nz = x.nonzero()
         self.assertFalse(nz.requires_grad)
 
+class TestShapeFuncs(TestCase):
+    @dtypes(*(torch.uint8, torch.int64, torch.double, torch.complex128))
+    @ops([op for op in shape_funcs if op.name in ['tile', 'repeat']])
```

Comment:
This is a good way to filter the OpInfos. In the future we may want to consider allowing filtering directly by name or function. That is, 

```
@ops('tile', 'repeat')
@ops(torch.tile, torch.repeat)
```

Which might be simpler to remember and more readable.